backend/src/models/CampusTutorModel.ts

```
@@ -0,0 +1,231 @@
+  availability: string[];
+}
+
+export interface BookingSession {
+  id: string;
+  tutorId: string;
+  tutorName: string;
+  courseCode: string;
+  studentName: string;
+  scheduledTime: string;
+  durationMinutes: number;
+  sessionType: 'one-on-one' | 'group' | 'exam-prep';
```

ESLint

[error] Replace `'one-on-one'·|·'group'·|·'exam-prep'` with `"one-on-one"·|·"group"·|·"exam-prep"`

(prettier/prettier)